package.json

```diff
@@ -0,0 +1,44 @@
+{
+  "name": "code_compete",
+  "version": "1.0.0",
+  "description": "Coding Competition with TDD and TypeScript. \"Code Compete\" team will use this repository for practicing best practices while solving algorithmic challenges. Every day, a new challenge will be introduced to the team, team has to solve the challenge and commit their solution to this repository by making a PR to the master branch.",
+  "main": "jest.config.js",
+  "dependencies": {
```

Comment:
The dependencies jest, ts-jest and typescript can be moved in devDependencies.

Reply:
Thank you Pareen for this good suggestion. I will implement this now.